asset/cloud-client/quickstart_exportassets_test.py

```diff
@@ -44,9 +51,35 @@ def asset_bucket(storage_client):
         raise e
 
 
-def test_export_assets(asset_bucket, capsys):
+@pytest.fixture(scope='module')
+def dataset(bigquery_client):
+    dataset_id = "{}.{}".format(PROJECT, DATASET)
+    dataset = bigquery.Dataset(dataset_id)
+    dataset.location = "US"
+    try:
+        dataset = bigquery_client.create_dataset(dataset)
+    except Exception as e:
+        print("Failed to Create dataset {}".format(dataset_id))
```

Comment:
Is this necessary? I'd hope that the dataset ID would be present in the stacktrace.